### PR TITLE
Add test module for man pages

### PR DESCRIPTION
**Pull request by cedvid**
- Related ticket: https://progress.opensuse.org/issues/156424
- Verification run (please disregard unrelated failures): 
  - SLE https://openqa.suse.de/tests/overview?build=os-autoinst%2Fos-autoinst-distri-opensuse%2318866
  - Leap/TW https://openqa.opensuse.org/tests/overview?build=os-autoinst%2Fos-autoinst-distri-opensuse%2318866

### CODE REVIEW

File: tests/console/man_pages.pm [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+use serial_terminal 'select_serial_terminal';
+use strict;
+use warnings;
+use utils 'zypper_call';
+
+sub run {
+    my ($self) = @_;
+    select_serial_terminal;
+
+    if (script_run 'rpm -q man-pages man') {
```
Comment (rfan1) [inline tests/console/man_pages.pm:21]: Is there any reason to add this test? 

if no, we can use single line `zypper_call 'in man-pages man';`

**Comment by rfan1** [comment]
Several questions:
1. Do you have any MR to load this test on qam?
2. Do you plan to add the test to leap and tw?
3. Can you please run some VRs on other sle versions if any?

File: tests/console/man_pages.pm (hunk 1 above)
Comment (cedvid) [reply tests/console/man_pages.pm]: Noted. I changed it.

**Comment by cedvid** [comment]
Hi @rfan1, 
1. I added the test to existing schedules (`mau_extratests1` for sle and `extra_tests_textmode` for leap/TW/15sp6)
2. Yes, see above
3. Done :)

File: schedule/functional/extra_tests_textmode.yaml [hunk 2]
```diff
@@ -97,6 +97,7 @@ schedule:
     - console/system_prepare
     - console/prepare_test_data
     - console/consoletest_setup
+    - console/man_pages
```
Comment (dzedro) [inline schedule/functional/extra_tests_textmode.yaml:100]: Please move the test at least after `'{{update_repos}}'`

**Review by dzedro — APPROVED** [review]

**Review by rfan1 — APPROVED** [review]

**Pull request merged**